frontend/tests/app/index.js

```diff
@@ -14,7 +15,14 @@ describe('Logged out state', () => {
       Object.defineProperty(Element.prototype, 'innerHTML', {
         set: function(value) {
           // u been a baaaaahd boy
-          calledInnerHTML++;
+          if (!['test', 'suite'].some(val => value.includes(val))) {
```

Comment:
Is this excluding the invocations from mocha? if so then why are there still 15 calls that we have to account for?